docs/logging.md

```diff
@@ -0,0 +1,58 @@
+# Logging
+
+Using Bigflow, you can easily add handler to your root logger that will send all loggers calls to the Cloud Logging.
```

Comment:
Using Bigflow, you can easily add a handler to your root logger that will send all logging calls to the Cloud Logging.